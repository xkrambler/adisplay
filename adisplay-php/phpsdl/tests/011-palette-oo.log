
---- EXPECTED OUTPUT
= Alloc
Parameter 'ncolors' is invalid
object(SDL_Palette)#%d (4) {
  ["ncolors"]=>
  int(1)
  ["version"]=>
  int(1)
  ["refcount"]=>
  int(1)
  ["colors"]=>
  array(1) {
    [0]=>
    object(SDL_Color)#%d (4) {
      ["r"]=>
      int(255)
      ["g"]=>
      int(255)
      ["b"]=>
      int(255)
      ["a"]=>
      int(255)
    }
  }
}
= Color
object(SDL_Color)#%d (4) {
  ["r"]=>
  int(255)
  ["g"]=>
  int(255)
  ["b"]=>
  int(255)
  ["a"]=>
  int(255)
}

%s: SDL_Palette::SetColors(): Invalid first color index, 2 >= 1 in %s/011-palette-oo.php on line 13
int(-1)
int(0)
object(SDL_Color)#%d (4) {
  ["r"]=>
  int(1)
  ["g"]=>
  int(2)
  ["b"]=>
  int(3)
  ["a"]=>
  int(4)
}
= Array
int(1)
object(SDL_Color)#%d (4) {
  ["r"]=>
  int(7)
  ["g"]=>
  int(8)
  ["b"]=>
  int(9)
  ["a"]=>
  int(0)
}
object(SDL_Color)#%d (4) {
  ["r"]=>
  int(0)
  ["g"]=>
  int(0)
  ["b"]=>
  int(0)
  ["a"]=>
  int(0)
}
= Free
= Done
---- ACTUAL OUTPUT
= Alloc
Parameter 'ncolors' is invalid
/usr/bin/php7.0: symbol lookup error: /data/dev/accord_display8th/adisplay/phpsdl/modules/sdl.so: undefined symbol: zend_objects_get_address
---- FAILED
