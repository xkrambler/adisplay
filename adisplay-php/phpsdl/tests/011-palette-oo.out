= Alloc
Parameter 'ncolors' is invalid
/usr/bin/php7.0: symbol lookup error: /data/dev/accord_display8th/adisplay/phpsdl/modules/sdl.so: undefined symbol: zend_objects_get_address